docs/.vuepress/theme/components/SearchBox.vue

```diff
@@ -1,35 +1,54 @@
 <template>
-  <div class="search-box">
-    <icon name="search" />
-    <input
-      @input="query = $event.target.value"
-      aria-label="Search"
-      :value="query"
-      :class="{ focused: focused }"
-      autocomplete="off"
-      spellcheck="false"
-      @focus="focused = true"
-      @blur="focused = false"
-      @keyup.enter="go(focusIndex)"
-      @keyup.up="onUp"
-      @keyup.down="onDown"
-    />
+  <div :class="searchClasses">
+    <h1 class="search-title">
+      <icon
+        name="chevron-right"
+        class="icon-back"
+        @click.native="$emit('search-toggle', false)"
+      />
+      Search
+    </h1>
+
+    <div class="search-bar">
+      <input
+        id="main-search-field"
+        @input="query = $event.target.value"
+        aria-label="Search"
+        :value="query"
+        autocomplete="off"
+        spellcheck="false"
+        @focus="$emit('focus-toggle', true)"
+        @blur="$emit('focus-toggle', false)"
+        @keyup.enter="go(focusIndex)"
+        @keyup.up="onUp"
+        @keyup.down="onDown"
+        placeholder="Search"
+      />
+      <icon name="search" class="icon-search-field" />
+    </div>
+
+    <div v-if="blankState && isSearchFocused" class="blank-state">
+      <div class="blank-state-emoji">{{ blankState.emoji }}</div>
+      <span>{{ blankState.text }}</span>
+    </div>
+
+    <h2 v-if="!blankState" class="results-title">Results</h2>
     <ul
+      v-if="!blankState && isSearchFocused"
       class="suggestions"
-      v-if="showSuggestions"
       :class="{ 'align-right': alignRight }"
-      @mouseleave="unfocus"
     >
       <li
         class="suggestion"
         v-for="(s, i) in suggestions"
         :class="{ focused: i === focusIndex }"
```

Comment:
You deleted the `.focused` class, so you can't see which search result is focused.

Current:
![Image 2020-03-05 at 7 45 44 AM](https://user-images.githubusercontent.com/8097623/75998227-5b2d2c00-5eb5-11ea-97cf-71625b206a16.png)

Your version:
![Image 2020-03-05 at 7 45 24 AM](https://user-images.githubusercontent.com/8097623/75998254-641dfd80-5eb5-11ea-889c-6755bfb194ca.png)

Let's bring this back :)

Reply:
This is resolved — now using :focus-within instead of JS